firebase/README.md

```diff
@@ -39,19 +81,32 @@ $ npm install -g firebase-tools
 
 Then, using you kiosk account UID from the previous step, along with a Slack legacy token which can be generated [here](https://api.slack.com/custom-integrations/legacy-tokens#legacy-info) run the following 2 commands:
 
+You will also need to run:
+```
+{HONESTY_DIR}\functions $ npm install
+{HONESTY_DIR}\functions $ firebase use {PROJECT_NAME}
+```
+
+to initialise the firebase-functions module and set up your target
+
+Now run the following
+
 ```shell
-$ firebase functions:config:set honestystore.uid=KIOSK_ACCOUNT_UID
-$ firebase functions:config:set slack.token=SLACK_TOKEN
+{HONESTY_DIR}\functions $ firebase functions:config:set honestystore.uid=KIOSK_ACCOUNT_UID
+{HONESTY_DIR}\functions $ firebase functions:config:set slack.token=SLACK_TOKEN
+{HONESTY_DIR}\functions $ firebase functions:config:set slack.storageurl=SLACK_STORAGE_URL
```

Comment:
Can we change the name of this to `snackchat.storageUrl`? I think the `slack` bit is confusing but maybe it was a typo?

Reply:
It was designed to keep the two related items together. (As the snackchats are stored only to submit to Slack). Are they actually more conceptually distinct than that?)

Function config items have to be all lowercase, so that's why it's not storageUrl.